apps/src/templates/ProjectTemplateWorkspaceIcon.jsx

```diff
@@ -28,6 +33,7 @@ export default class ProjectTemplateWorkspaceIcon extends React.Component {
           role="tooltip"
```

Comment:
Currently, the icon/tooltip is not tab-navigable. While you're here, what do you think of wrapping image in a button? With the version of `ReactTooltip` we're using `3.2.7` (kinda old), you could add the prop `data-event`. Here's an example: https://github.com/code-dot-org/code-dot-org/commit/fa62bd24c24efc9b80ace5b25118986c4b90f35d

Reply:
Nit: could migrate style to module if you make suggested update above.

Reply:
Will do, thanks for the code snippet!